engine/core/src/main/java/org/datacleaner/metamodel/datahub/DataHubSecurityMode.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.metamodel.datahub;
+
+/**
+ * Enumerates the supported DataHub security models that are supported by the
+ * DataHub Connector.
+ *
+ */
+public enum DataHubSecurityMode {
+    DEFAULT, CAS;
```

Comment:
What is default? Is that HTTP BASIC authentication? As a user this could confuse me a bit - I would prefer to have the concrete options listed and not a "default" unless that default is really unambigious. I guess most DH customers would not know if they have a non-default security mode.